src/components/LogIn.jsx

```diff
@@ -17,10 +18,20 @@ const formErrors = {
 }
 
 function LogIn() {
-
     const [loginValues, setLoginValues] = useState(formValues);
     const [loginErrors, setLoginErrors] = useState(formErrors);
 
+    const history = useHistory();
+
+    const onSubmit = () => {
+        axios.post('https://reciplease-application.herokuapp.com/users/login', loginValues)
+            .then(res => {
+                console.log('congratulations you fuck, welcum to our website af', res)
+                history.push('/search')
+            })
+            .catch(err => {console.log(err)})
+    }
+
```

Comment:
I think this is really good. The app is in a place where we can use Redux as our global state management system. This means we can have the submit event take place in actions and keep track of user information while logged in as state.

Reply:
After speaking with Austin, we have agreed that loginValues does not need to be a globally accessed piece of state, since the database will provide user credentials. However, we do need global state for sessions.